detector/aws_instance_invalid_type.go

```diff
@@ -9,22 +9,21 @@ import (
 
 type AwsInstanceInvalidTypeDetector struct {
 	*Detector
-	IssueType     string
-	TargetType    string
-	Target        string
-	DeepCheck     bool
 	instanceTypes map[string]bool
 }
 
 func (d *Detector) CreateAwsInstanceInvalidTypeDetector() *AwsInstanceInvalidTypeDetector {
-	return &AwsInstanceInvalidTypeDetector{
+	nd := &AwsInstanceInvalidTypeDetector{
 		Detector:      d,
-		IssueType:     issue.ERROR,
-		TargetType:    "resource",
-		Target:        "aws_instance",
-		DeepCheck:     false,
 		instanceTypes: map[string]bool{},
 	}
+	nd.Name = "aws_instance_invalid_type"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_instance"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_instance" found in: aws_instance_not_specified_iam_profile.go:17:14 aws_instance_invalid_subnet.go:23:14 aws_instance_previous_type.go:23:14 aws_instance_invalid_key_name.go:23:14 aws_instance_invalid_ami.go:23:14 aws_instance_invalid_vpc_security_group.go:24:14 aws_instance_invalid_iam_profile.go:23:14 aws_instance_default_standard_volume.go:17:14

<!-- Comment by [SideCI](https://sideci.com) -->